kyve/chain.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "kyve",
+  "chain_id": "kyve-1",
+  "pretty_name": "KYVE",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "kyve",
```

Comment:
Please have this prefix registered to https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Just submitted a PR - https://github.com/satoshilabs/slips/pull/1512

Reply:
Looks like they move fast - already merged 🔥